package.json

```diff
@@ -9,7 +9,7 @@
   "homepage": ".",
   "scripts": {
     "dev": "vite --open",
-    "dev:mock": "MOCK=1 vite --open",
+    "dev:mock": "MOCK=0 vite --open",
```

Comment:
@TahaKhanAbdalli - it would be more logical for `MOCK` to take value of `1`.  `0` tends to equal `False`.

Do we need to assign value to `MOCK`?  Can't we just have `MOCK` in command-line?  When we decide the backend, we know to use `REST` if mock is falsy.

Reply:
> @TahaKhanAbdalli - it would be more logical for `MOCK` to take value of `1`. `0` tends to equal `False`.
> 
> Do we need to assign value to `MOCK`? Can't we just have `MOCK` in command-line? When we decide the backend, we know to use `REST` if mock is falsy.

Now only need to set `MOCK` from command line no matter the value and run `yarn run dev`, this will run `MOCK` and to run `REST`, we don't need to set `MOCK` from command line.